.changeset/late-buttons-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/theme": patch
+---
+
+WHAT: Used `@ts-ignore` to bypass type checking, Functions used `Record<string, string>` or `Object`, causing potential type issues and limited flexibility. Functions now use `UnknownMap` type for better type safety and flexibility.
```

Comment:
Thank you for your feedback. Initially, I used `unknown` for the value type to provide flexibility, as the values in the object could be strings, numbers, or other types. However, given the current context, I believe it would be fine to declare the value type as `string`.

Reply:
@wingkwong It's corrected. And I changed the value to `string` instead of `unknown` type.